tests/test-team/template-mgmt-component-tests/template-mgmt-create-nhs-app-template-page.component.ts

```diff
@@ -43,4 +55,253 @@ test.describe('Create NHS App Template Page', () => {
       'Create NHS App message template'
     );
   });
+
+  test('Validate error messages on the create NHS App message template page with no template name or body', async ({
+    page,
+    baseURL,
+  }) => {
+    const createTemplatePage = new TemplateMgmtCreatePage(page);
+
+    await createTemplatePage.navigateToCreateNhsAppTemplatePage(
+      nhsAppNoTemplateSessionData.id
+    );
+
+    await expect(page).toHaveURL(
+      `${baseURL}/templates/create-nhs-app-template/${nhsAppNoTemplateSessionData.id}`
+    );
+    expect(await createTemplatePage.pageHeader.textContent()).toBe(
+      'Create NHS App message template'
+    );
+    await createTemplatePage.clickContinueButton();
+    await expect(page.locator('.nhsuk-error-summary')).toBeVisible();
+    await expect(page.locator('.nhsuk-error-summary')).toHaveText([
+      'There is a problemEnter a template nameEnter a template message',
```

Comment:
The test should demonstrate that these appear as separate messages, not a concatenation.

Reply:
As with similar examples below.

Reply:
Updating this so they are no longer this.